lib/overcommit/hook/post_commit/git_guilt.rb

```diff
@@ -0,0 +1,9 @@
+module Overcommit::Hook::PostCommit
+  # Calculates the change in blame since the last revision.
+  class GitGuilt < Base
+    def run
+      system(executable, 'HEAD~', 'HEAD')
```

Comment:
I originally wrote it this way:
```ruby
def run
  result = execute([executable, 'HEAD~', 'HEAD'])
  puts result.stdout
  :pass
end
```
But `execute` seems to strip all formatting (and newlines, weirdly) from `result.stdout`. This was the only way I could get the correctly formatted and colored output.